### PR TITLE
Remove Tracer

### PR DESCRIPTION
**Pull request by lukas-phaf**
Remove tracer. Clean up proj cache on program exit. Fix memory leak.

### CODE REVIEW

File: Docker/docker-compose.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ services:
         max-file: "10"
   adaguc-server:
     image: openearth/adaguc-server
+#    build: ..
```
Comment (lukas-phaf) [inline Docker/docker-compose.yml]: Revert changes on this file.

File: adagucserverEC/adagucserver.cpp [hunk 2]
```diff
@@ -390,6 +394,35 @@ int _main(int argc, char **argv, char **) {
 }
 
 int main(int argc, char **argv, char **envp) {
+  clock_t start = clock();
```
Comment (lukas-phaf) [inline adagucserverEC/adagucserver.cpp]: Remove timing and memory error code.

File: hclasses/CDebugger_H.h [hunk 3]
```diff
@@ -33,54 +33,7 @@
...
-private:
-  void lock() { _lockCount++; }
-  void unlock() { _lockCount--; }
-
-private:
-  std::map<void *, Entry> _map;
-  int _lockCount;
-};
-
+// TODO: This file is now empty...
```
Comment (lukas-phaf) [inline hclasses/CDebugger_H.h]: Actually, it still defines 4 functions

Comment (maartenplieger) [reply hclasses/CDebugger_H.h]: Not true! ;D

**Review by maartenplieger — APPROVED** [review]

**Pull request merged**